e2e-tests/admin/tests/worker-tags.spec.js

```diff
@@ -67,9 +66,7 @@ test('Worker Tags @ce @ent @aws @docker', async ({ page }) => {
       .getByRole('columnheader')
       .nth(i)
       .innerText();
-    if (header == 'Tag') {
-      dialogTagIndex = i;
```

Comment:
Not sure if this was previously used and is vestigial but some of these variables weren't used at all and tests ran fine without them